src/libs/OptionsListUtils.ts

```diff
@@ -1588,7 +1587,8 @@ function orderOptions(options: ReportUtils.OptionData[], searchValue: string | u
                 if (preferChatroomsOverThreads && option.isThread) {
                     return 4;
                 }
-                if (!!option.isChatRoom || option.isArchivedRoom) {
+                const reportNameValuePairs = ReportUtils.getReportNameValuePairs(option?.reportID);
+                if (!!option.isChatRoom || ReportUtils.isArchivedRoom(option, reportNameValuePairs)) {
```

Comment:
Can we use `option.private_isArchived` here as well? OptionData extends the Report data type which has `private_isArchived`